turbo.json

```diff
@@ -11,20 +11,61 @@
     },
     "build": {
       "dependsOn": ["^build"],
-      "cache": false,
-      "outputs": ["dist/**"]
+      "cache": true,
+      "inputs": [
+        "src/**",
+        "app/**",
+        "public/**",
+        "index.*",
+        "tsconfig.json",
+        "tsconfig.*.json",
+        "package.json",
+        "next.config.*",
+        "vite.config.*"
+      ],
+      "outputs": ["dist/**", "lib/**", ".next/**", "!.next/cache/**"]
     },
     "typecheck": {
-      "cache": false
+      "cache": true,
+      "dependsOn": ["^build"],
+      "inputs": [
+        "src/**",
+        "app/**",
+        "index.*",
+        "tsconfig.json",
+        "tsconfig.*.json",
+        "package.json",
+        "next.config.*",
+        "vite.config.*"
+      ]
     },
     "test": {
       "cache": false
     },
     "lint": {
-      "cache": false
+      "cache": true,
+      "inputs": [
+        "src/**",
```

Comment:
We dont have this structure. Is this meant for lab instead?

Reply:
the turbo.json configuration is correctly set up for your monorepo structure. The seemingly generic paths (src/**, app/**) are actually specific to each package/app because they are resolved relative to the directory where Turborepo executes the task for that particular workspace.